docs/source/configuration.rst

```diff
@@ -212,30 +209,29 @@ be a :code:`.sql` itself.
     be a requirement that the *templating* is accurate - only so far as that
     is required to enable the *parsing* and *linting* to be helpful.
 
-Builtin Macro Blocks
```

Comment:
We'll need to deprecate the builtin macro blocks related to DBT

Reply:
https://github.com/sqlfluff/sqlfluff/issues/554